app/templates/faq.html

```diff
@@ -39,5 +39,9 @@ <h4>Removing information from the Yale Directory</h4>
     <p>Because directory information is generally public record as a matter of FERPA policy, Yale does not have an interactive website that allows you to remove it. If you would like your information removed, Yale recommends that you contact <a href="mailto:registrar@yale.edu">registrar@yale.edu</a> to request removal. Bear in mind this may prevent you from using some university services.</p>
     <h3>Q: How can I add a pronunciation for my name?</h3>
     <p>Yalies draws name pronunciations from <a href="https://name-coach.com">NameCoach</a>, a third-party company that Yale uses to provide name pronunciations, plus additional information like phonetic spelling and pronouns. You can provide this information to Yale by visiting <a href="https://www.sis.yale.edu">Yale SIS</a> and selecting "Personal Data" then "NameCoach".</p>
+    <h3>Q: I can't log in to Yalies, how do I re-gain access?</h3>
+    <p>You need to add yourself back to the directory of students at <a href="https://directory.yale.edu">the Yale directory</a> You can do so in your <a href="https://yub.yale.edu">Yale Hub</a> account under the personal data tab, directory listing option.</p>
```

Comment:
Super nit, seems there's a missing period after the first link. Also maybe we can include screenshots?

Reply:
Pushing changes and images to this branch